sdk/nodejs/x/automation/stack.ts

```diff
@@ -174,16 +214,36 @@ export class Stack {
         }
 
         args.push("--exec-kind", kind);
+
+        let logPromise;
```

Comment:
Probably should put a type annotation on this?

Reply:
Huh, my IDE usually calls me out on this stuff 🤔